scripts/make_src_archive.sh

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env bash
 
-set -eu
+set -eux
 
-OUTPUT=$1
-REPO=${2:-.}
+SCRIPTS_DIR=$(readlink -f "$(dirname "$0")")
+
+OUTPUT=$(realpath "$1")
+REPO=$(realpath "${2:-.}")
 
 python3 -m venv venv
 
 . venv/bin/activate
 
-pip install 'git_archive_all==1.19.4'
-
-ABS_OUT=$(realpath "$OUTPUT")
+# Use our fork which makes it easier to add extra files
+# pip install 'git_archive_all==1.19.4'
+pip install git+https://github.com/lbonn/git-archive-all.git
```

Comment:
Is this something that you can upstream? Alternatively would be better to have in the same namespace (advancedtelematic).

Reply:
- Yes https://github.com/Kentzo/git-archive-all/pull/73
- ok I can do that. Initially thought it was not worth the noise

Reply:
> ok I can do that. Initially thought it was not worth the noise

Nah, don't worry about it for now if you've already got the PR to merge upstream. It's fine for now.